frontend/app/src/store/__test__/main.spec.ts

```diff
@@ -934,47 +934,47 @@ describe('Main store', () => {
     expect(store.exportMetricsBountiesCsv).toBeCalledWith(body);
   });
 
-  it("I should be able to test that the signed-in user details are persisted in the local storage", async () => {
+  it('I should be able to test that the signed-in user details are persisted in the local storage', async () => {
     uiStore.setShowSignIn(true);
 
     localStorageMock.setItem('ui', JSON.stringify(uiStore));

```

Comment:
@ManyRios Mock the user info, don't use the empty uiStore, so you can test for equality between the mocked user and that saved to local storage